samza-api/src/main/java/org/apache/samza/table/descriptors/LocalTableDescriptor.java

```diff
@@ -67,6 +67,16 @@ public LocalTableDescriptor(String tableId, KVSerde<K, V> serde) {
     this.serde = serde;
   }
 
+  /**
+   * Add side inputs to the table. Each stream is of the format
+   * <i>system-name</i>.<i>stream-name</i>. The streams are marked as bootstrap streams and once the table is bootstrapped, it is
+   * updated on the background in change capture mode.
+   * Applications should specify the transformation logic using {@link #withSideInputsProcessor(SideInputsProcessor)}, which is
```

Comment:
"which will be applied to the incoming messages and the results will be written to the table."